### ISSUE TITLE
W2 Ch10

### ISSUE THREAD

**Issue by JSchuurmans** [JannesKlaas/MLiFC#12]
Everything looks good like we would have expected it. (good like -> good, like / like we would have expected / like we would have expected it to be.)

Train / Dev / Test:
Good and important part. I would add the following: We evaluate on a neutral test set because we want to see the performance of our model out of sample. The out of sample performance can be approximated by running the model on a unseen dataset. We would like to know the out of sample performance because this indicates whether the model generalizes well. In other words, to see if we are not overfitting the model to the training or dev set. (fitting the noise instead of the signal)

In [7]: Using TensorFlow backend. 'Couldn't import dot_parser, loading of dot files will not be possible.'

The baseline model:
Good practice to compare with a baseline model, but I still don't like the sound of log loss.

The ReLu activation function:
'... converges faster (see Krizhevsky et al..' -> add closing bracket

....both tanh and relu (in green)... -> for me they are blue and red.

Mathematically: W=W-alpha*V_{dW}. A typical value for beta... (do you mean alpha or beta?)

L1 Regularization:
The difference is the marginal penalty when W is very small or very large. In practice the difference is not very large.

???Batchnorm in combination with normalization, what is the effect of using both vs just one of the two???
